packages/electron-app/src/main/index.ts

```diff
@@ -2,23 +2,85 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { ChildProcess } from 'child_process';
 import electron from 'electron';
-import installExtension, { REACT_DEVELOPER_TOOLS } from 'electron-devtools-installer';
-import Pino from 'pino';
 import path from 'path';
+import Pino from 'pino';
 import url from 'url';
 
-import { CSP, staticPath } from './util';
+import { CSP, isSubstrateRunning, runSubstrateDev, staticPath } from './util';
 
 const { app, BrowserWindow, session } = electron;
-const pino = Pino();
-let mainWindow: Electron.BrowserWindow | undefined;
+const pino = new Pino();
+let sluiApp: Electron.BrowserWindow | undefined;
+let substrateProc: ChildProcess;
```

Comment:
You can put this variable into substrateProcess.ts, 1/ makes more sense imo and 2/ you don't need setProc anymore